includes/class-wi-volunteer-management.php

```diff
@@ -81,6 +81,7 @@ public function __construct() {
 
 		// Load the public hooks
 		$this->define_public_hooks();
+      $this->define_widget_hooks();
```

Comment:
I like that you broke this out, but since all the other hooks are defined within `define_public_hooks()` and `define_admin_hooks()` admin hooks I'd say let's just put the code within `define_public_hooks()` instead of creating a new `define_widget_hooks()` function.